lib/vdc/BOVCollection.cpp

```diff
@@ -123,8 +124,13 @@ int BOVCollection::Initialize(const std::vector<std::string> &paths)
             if (_dataFile == _defaultFile) { return _missingValueError(DATA_FILE_TOKEN); }
             if (_dataFormat == _defaultFormat) { return _missingValueError(FORMAT_TOKEN); }
             if (_gridSize == _defaultGridSize) { return _missingValueError(GRID_SIZE_TOKEN); }
+            if (_time == _defaultTime) { return _missingValueError(TIME_TOKEN); }
 
-            _populateDataFileMap();
+            rc = _populateDataFileMap();
```

Comment:
It would be better to have the error message inside of _populateDataFileMap() for a couple of reasons:

1. Our error reporting convention states that functions that return a negative int on failure should report an error with SetErrMsg() (prefer a boolean return status for functions that do NOT report a message).
2. Better future proofing: if some other error condition gets added to _populateDataFileMap() later the current error message may not make sense.